packages/jest-watch/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "jest-watch",
+  "description": "Delightful JavaScript Testing.",
+  "version": "23.0.1",
+  "main": "build/index.js",
+  "dependencies": {
+    "ansi-escapes": "^3.0.0",
+    "chalk": "^2.0.1",
+    "string-length": "^2.0.0"
```

Comment:
Please make sure these are all deps `jest-watch` use

Reply:
I copied `jest-cli` and removed unused stuff, so it should be enough

Reply:
haven't checked if I can remove anything from jest-cli, though, lemme do that